.github/README.md

```diff
@@ -12,99 +12,194 @@ Welcome to **Backdrop**, a command-line utility designed to manage your desktop
+3. Open "**Edit the System Environment Variables**".
+4. Go to the "**Advanced**" tab and click "**Environment Variables**".
+5. Edit the **Path** variable under System Variables and add the folder where the `backdrop.exe` binary is located. (Do not include the binary name.)
+
+</details>
+
+### Install From Source Code
+
+<details>
+<summary>Steps to Build and Install From Source</summary>
+
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/JanMichaelSE/backdrop.git
+   cd backdrop
+   ```
+2. Build the project:
+   ```bash
+   go build
+   ```
+3. Move the binary to a directory in your `PATH`:
+   ```bash
+   mv backdrop $HOME/.backdrop/bin
+   ```
+4. Add the binary to your `PATH`:
+   ```bash
+   echo 'export PATH=$HOME/.backdrop/bin:$PATH' >> $HOME/.bashrc
+   source $HOME/.bashrc
+   ```
+   </details>
+
+---
 
-Optionally move it to a directory in your PATH.
-```bash
-mv backdrop $HOME/.backdrop/bin
-```
+## :wastebasket: Uninstall
 
-Finally add the binary into the `PATH` variable
-```bash
-echo 'export PATH=$HOME/.backdrop/bin:$PATH' >> $HOME/.bashrc
-source $HOME/.bashrc
-```
+### Uninstall Using Go
 
-## :wastebasket: Uninstall
+<details>
+<summary>Steps to Uninstall</summary>
 
 ```bash
 rm $(go env GOPATH)/bin/backdrop
-go clean -modcache # <- Optional
+go clean -modcache  # Optional
 ```
 
+</details>
+
+### Uninstall on Linux
+
+<details>
+<summary>Steps to Remove Installed Binary</summary>
+
+1. Locate the binary:
+   ```bash
+   which backdrop
+   ```
+2. Delete the binary:
+   ```bash
+   rm /path/to/backdrop
+   ```
+3. (Optional) Remove configuration files, if applicable:
+   ```bash
+   rm -rf $HOME/.backdrop
+   ```
+4. Edit ~/.bashrc and remove the line that adds Backdrop to PATH
+   ```bash
+   nano ~/.bashrc
+   ```
+   Remove the following line (or equivalent):
+   ```bash
+   export PATH=$HOME/.backdrop/bin:$PATH
+   ```
+
+</details>
+  
+### Uninstall on Windows
+
+<details>
+<summary>Steps to Remove Installed Binary</summary>
+
+1. Open a Command Prompt and verify the location of the binary:
+   ```powershell
+   where backdrop
+   ```
+2. Remove the folder path from your PATH environment variable:
+    - Navigate to Start Menu and search for "Environment Variables".
+    - Open Edit the System Environment Variables.
+    - In the Advanced tab, click Environment Variables.
+    - Under the System Variables, select Path and click Edit.
+    - Remove the folder containing backdrop.exe from the list.
+</details>
+
+---
+
 ## &#x2705; What's Supported
 
-#### Operating Systems
-- Ubuntu/GNOME based Distros
-- CentOS/MATE (Doesn't support Slideshows)
-- Coming Soon:
-    - MacOS
-    - Microsoft (Make a PR because I won't do it)
-- Will not be Supported:
-    - WSL
+### Operating Systems
+
+- **Supported**:
+  - Ubuntu/GNOME-based Distros
+  - CentOS/MATE (Slideshows are not supported)
+  - Windows (Slideshows are not supported)
+- **Coming Soon**:
+  - macOS
```

Comment:
This should also say `Make a PR because I won't do it` HAHAAHAHA

Reply:
Added comment.